app/src/main/java/com/example/tungcon/demo/data/source/RemoteDataSource.java

```diff
@@ -0,0 +1,7 @@
+package com.example.tungcon.demo.data.source;
+
+import com.example.tungcon.demo.model.Movie;
+
+public interface RemoteDataSource {
```

Comment:
e nên đặt tên interface này là 
```suggestion
public interface MovieDataSource {
```
vì nó chịu trách nhiệm tuơng tác với model Movie